api/src/main/resources/rdr.yaml

```diff
@@ -325,6 +325,8 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/AccessTier'
+      demographicSurveyV2:
```

Comment:
changes in this file are from the prototype.

not strictly needed for this story, but this will be useful for RW-8355.

This PR **does not** populate the V2 Demo Survey in the RDR Export.